lib/core/interfaces.ts

```diff
@@ -1,4 +1,4 @@
-import { Server, IRouteConfiguration } from 'hapi'
+import { Server, RouteConfiguration } from 'hapi'
```

Comment:
`RouteConfiguration` could be removed, not used anymore in this file.
I will remove it during the merge process.